ci/test.yml

```diff
@@ -79,7 +79,7 @@ core monero test:
   variables:
     TREZOR_PROFILING: 1
   script:
-    - nix-shell --run "poetry run make -C core test_emu_monero"
+    - nix-shell --arg fullDeps true --run "poetry run make -C core test_emu_monero"
```

Comment:
After merging, other PRs need to be rebased on top of this one because of this line I think. Considering workarounds.

Reply:
We can live with that IMHO.